internal/syncasync/sync_async_bridge.go

```diff
@@ -22,57 +22,67 @@ import (
 	"time"
 
 	"github.com/hyperledger-labs/firefly/internal/data"
-	"github.com/hyperledger-labs/firefly/internal/events"
 	"github.com/hyperledger-labs/firefly/internal/i18n"
 	"github.com/hyperledger-labs/firefly/internal/log"
-	"github.com/hyperledger-labs/firefly/internal/privatemessaging"
+	"github.com/hyperledger-labs/firefly/internal/sysmessaging"
 	"github.com/hyperledger-labs/firefly/pkg/database"
 	"github.com/hyperledger-labs/firefly/pkg/fftypes"
 )
 
 // Bridge translates synchronous (HTTP API) calls, into asynchronously sending a
 // message and blocking until a correlating response is received, or we hit a timeout.
 type Bridge interface {
+	// Init is required as there's a bi-directional relationship between private messaging and syncasync bridge
+	Init(sysevents sysmessaging.SystemEvents, sender sysmessaging.MessageSender)
```

Comment:
The circular dependency is maybe a point of concern... but I don't have an alternate suggestion at the moment, so probably just something to consider down the line if it can be improved.